app/helpers/bootstrap_helper.rb

```diff
@@ -92,8 +92,8 @@ def pagination(collection)
   end
 
   # <div class="col-md-size form-group sqs-form"></div>
-  def form_group(&)
-    tag.div(class: 'form-group row sqs-form', &)
+  def form_group(classes = '', &)
```

Comment:
Would this be better making `classes = ' '` -> `class = 'form-group row sqs-form'` and then `:class` in the tag.div. Makes it fully overrideable.

Reply:
Yes, I guess it depends on how customisable we want it to be. Fine to leave as is 👍